apply/golden_file_test.go

```diff
@@ -119,100 +119,100 @@ func TestIntegration(t *testing.T) {
 	}
 }
 
-func TestIntegrationYaml(t *testing.T) {
```

Comment:
You shouldn't need to create a new go test, juts new test directories in `testdata`